administrator/components/com_guidedtours/src/Model/TourModel.php

```diff
@@ -481,4 +568,30 @@ protected function setStepsLanguage(int $id, string $language = '*'): bool
         return $db->setQuery($query)
             ->execute();
     }
-}
+
+    /**
+     * Method to change the alias when not unique.
+     *
+     * @param   string   $alias           The alias.
+     * @param   integer  $currentItemId   The id of the current tour.
+     *
+     * @return  string $alias  Contains the modified alias.
+     *
+     * @since   __VERSION_DEPLOY__
+     */
+    protected function generateNewAlias($alias, $currentItemId)
+    {
+        $unique = false;
+        // Alter the title & alias
+        while (!$unique) {
+            $aliasItem = $this->getItemByAlias($alias);
+            if ($aliasItem->id > 0 && $aliasItem->id != $currentItemId) {
+                $alias = StringHelper::increment($alias, 'dash');
+            } else {
+                $unique = true;
+            }
+        }
+
+        return $alias;
+    }
+}
```

Comment:
```suggestion
}

```
Last PHPCS fix. New line needed at the end of the file.

Reply:
Thanks Richard